### ISSUE TITLE
Neural Network I - Setup (Python)

### ISSUE THREAD

**Issue by BrandnerKasper** [BrandnerKasper/FusionEngine#17]
Find a suitable neural network to generate a PNG from an ASCII string in real-time.
We start with the simplest use case -> each character matches a pixel in the png.

Setup PyTorch:
- [x] dataloader
- [x] model (for now simple MLP?)
- [x] use CNN
- [x] use other encoding for input -> one hot vector
- [x] metrics (L1/PSNR?)
- [x] weighted L1
- [x] train
- [x] inference speed/VRAM usage
- [x] tensorboard visualization

Play around with: 
- [x] Batch Norm layers
- [x] Drop Out layers
